TestingScripts/Coverage/CoverageOpenCPP.py

```diff
@@ -0,0 +1,767 @@
+                methods = self.comps[c]['methods']
+                for m1 in methods:
+                    if file.lower() == methods[m1]['fileName'].lower() and line == methods[m1]['lineNo']:
+                        return m1
+        return None
+
+    #-------------------------------------------------------------------------------------------
+    # bsimethod
+    #-------------------------------------------------------------------------------------------
+    def methodsFromPDBs(self):
+        winSdk = os.getenv('Win10SdkDir')
+        if winSdk is None:
+            print ('Please set Win10SkDir environment variable for dbh.exe')
+            exit(-1)
+        else:
+            dbhPath = os.path.join(winSdk, 'Debuggers', 'x86', 'dbh.exe')
+            
+        if not os.path.exists(self.repoPath):
+            os.mkdir(self.repoPath)
+
+        #Gather method details for all components
+        for comp in self.comps:
+            methods = self.comps[comp]['methods']
+            pdbFile = cmp.PdbPathForComp(comp)
+            dumpFile = os.path.join(self.repoPath, comp+'_dump.txt')
+            if os.path.exists(pdbFile):
+                if not os.path.exists(dumpFile):
+                    print ('Generating dump for: ' + comp+'.pdb')
+                    cmd = '"' + dbhPath + '" '+ ' -c dump '+ pdbFile+ " > " + dumpFile
+                    print (cmd)
+                    result = subprocess.call(cmd, shell=True)
+                    if result is not 0:
+                        print ('Dump command failed: ' + cmd)
+                        exit(-1)
+                print ('Processing dump file: ' + comp+'_dump.txt')
+                with open(dumpFile, 'r') as f:
+                    lines = f.readlines()
+                    print(len(lines))
+                    for line in lines:
+                        if len(line) > 1:
+                            lineP = line.split('\t')
+                            method = lineP[1]
+                            if 'bvector<' in method.lower() or ('<' in method.lower() and '>' in method.lower()) or '::operator' in method.lower():
+                                continue
+                            fullFileName = lineP[len(lineP) - 3]
+                                          
+                            fileName = os.path.basename(fullFileName)
+                            fName, fExt = os.path.splitext(fileName)
+                            lineNo = int(lineP[len(lineP) - 2])
+                            skipMethod = False
+                            baseRoot = os.path.dirname(os.path.dirname(os.getenv('SrcRoot')))
+                            if fExt == ".h" and baseRoot.lower() in fullFileName.lower():
+                                with open(fullFileName, 'r') as f2:
+                                    lines2 = f2.readlines()
+                                    fullLine = lines2[lineNo - 1].strip()
+                                    if not '_EXPORT' in fullLine:
```

Comment:
removed

Reply:
But the code is still there, just commented for what seems like a future PR. I don't want someone to have to ask the same question in that future PR. A comment explaining why its necessary would be useful

Reply:
This part of the code needs looking into and review so I also added TODO comment on top of the method
If you think removing this code would be better and cleaner than commenting it. Then I can do that

Reply:
I'm fine with leaving the code commented out. I just want my questions to be captured into this code so that they don't need to be repeated on a future review. It can be as simple as TODO: Clarify why this line is needed

Reply:
Sure that can be done. Did the todo comment